temporalcli/commandsgen/commands.yml

```diff
@@ -3333,3 +3304,37 @@ option-sets:
         description: |
           Run ID.
           If unset, updates the currently-running Workflow Execution.
+
+  - name: nexus-endpoint-identity
+    description: Nexus endpoint commands
```

Comment:
Is this correct? Expecting something closer to "Commands related to the identity of the Nexus Endpoint". May be just not understanding this. Apologies if so

Reply:
Also, these are flags/options, not commands—so calling them commands is inaccurate.

Reply:
Also, shouldn't it be Nexus Endpoints?

Reply:
removing `description` [per comment above](https://github.com/temporalio/cli/pull/693/files#r1803621328)